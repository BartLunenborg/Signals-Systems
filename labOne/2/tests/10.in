4.0 -1.04719755
