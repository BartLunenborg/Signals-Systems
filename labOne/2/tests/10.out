2.00 -3.46
